lib/internal/webstreams/encoding.js

```diff
@@ -3,6 +3,9 @@
 const {
   ObjectDefineProperties,
   Symbol,
+  String,
+  StringPrototypeCharCodeAt,
```

Comment:
My bad! I mixed it up.
```suggestion
  String,
  StringPrototypeCharCodeAt,
  Symbol,
```

Reply:
4a92843565caa3efd903251c0b6fdfcad74c5e8c

Sorry to bother you again... It would be great if lint could automatically detect imports out of order.